R/mod_body_coc_competition.R

```diff
@@ -0,0 +1,475 @@
+            DQflag == 2 ~ "", # "Documents not yet received",
+            DQflag == 3 ~ "", # "Docs received, not yet scored",
+            DQflag == 4 ~ "", # "CoC Error",
+            DQflag == 5 ~ "" # "Docs received past the due date"
+            )
+          ) |>
+          dplyr::filter(!Measure %in%
+                   c("Long Term Homeless",
+                     "Prioritization of Chronic",
+                     "Prioritization Workgroup")) |>
+          dplyr::select(1, Calculation, 2, "Possible Score" = 4, "Data Quality" = DQ)
+        
+        th <- estimated_score |> dplyr::left_join(dq, by = "Measure") |>
+          dplyr::ungroup() |>
+          dplyr::left_join(possible_score, by = "Measure") |>
+          dplyr::left_join(calculation, by = "Measure") |>
+          dplyr::mutate(
+            DQ = dplyr::case_when(
+              DQflag == 1 ~ "Please correct your Data Quality issues so this item
+            can be scored",
+            DQflag == 0 ~ "Data Quality passes",
+            DQflag == 2 ~ "", # "Documents not yet received",
+            DQflag == 3 ~ "", # "Docs received, not yet scored",
+            DQflag == 4 ~ "", # "CoC Error",
+            DQflag == 5 ~ "" # "Docs received past the due date"
+            )
+          ) |>
+          dplyr::filter(!Measure %in% c(
+            "Long Term Homeless",
+            "Prioritization of Chronic",
+            "Prioritization Workgroup"
+          )) |>
+          dplyr::select(1, Calculation, 2, "Possible Score" = 4, "Data Quality" = DQ)
+
+        DT::datatable(
+          if (ptc == 3) {
+            psh
+          } else if (ptc == 13) {
+            rrh
+          } else if(ptc == 2) {
+            th
+          },
+          rownames = FALSE,
+          options = list(dom = 't',
+                         pageLength = 100)
+        )
+      })
+    
+    output$pe_ExitsToPH <- DT::renderDataTable({
+      a <- pe_exits_to_ph() |>
+        dplyr::filter(AltProjectName == input$pe_provider) |>
+        dplyr::mutate(MeetsObjective = dplyr::if_else(MeetsObjective == 1, "Yes", "No"),
+               Destination = living_situation(Destination)) |>
+        dplyr::select("Client ID" = PersonalID,
+               "Entry Date" = EntryDate,
+               "Move In Date" = MoveInDateAdjust,
+               "Exit Date" = ExitDate,
+               Destination,
+               "Destination Group" = DestinationGroup,
+               "Meets Objective" = MeetsObjective)    
+      
+      DT::datatable(a,
+                rownames = FALSE,
+                filter = 'top',
+                options = list(dom = 'ltpi'),
+                caption = "PSH: Heads of Household | 
+              TH, RRH: Heads of Household Leavers")
+      
+    })
+    
+    output$pe_BenefitsAtExit <- DT::renderDataTable({
+      a <- pe_benefits_at_exit() |>
+        dplyr::filter(AltProjectName == input$pe_provider) |>
+        dplyr::mutate(
+          BenefitsFromAnySource = dplyr::case_when(
```

Comment:
I think that `HMIS::hud_translations` has a function that takes care of this translation. Worth checking